src/scripts/add_models_to_leaderboard.py

```diff
@@ -0,0 +1,207 @@
+"""Script which searches the huggingface_hub for models fitting the supported tasks and add their results to the leaderboard."""
+
+import logging
+import os
+from csv import writer
+from typing import Any, Dict, List, Tuple
+
+import pandas as pd
+from huggingface_hub.hf_api import HfApi, ModelFilter
+
+from alexandra_ai_eval.evaluator import Evaluator
+from alexandra_ai_eval.task_configs import get_all_task_configs
+
+logger = logging.getLogger(__name__)
+
+
+def define_searches(task_mapping: Dict[str, Any]) -> List[Dict[str, Any]]:
+    """Define the searches to be performed on the huggingface_hub.
```

Comment:
It's quite difficult to understand what this is doing. Maybe give a slightly more detailed description below this short snippet? What are "searches"? How does the task mapping play into it?